spec/numberformat.html

```diff
@@ -1351,7 +1359,7 @@ <h1>
           1. Let _int_ be the length of _a_.
         1. Else,
           1. Let _int_ be the length of _m_.
-        1. Let _cut_ be _maxFraction_ - _minFraction_.
+        1. Let _cut_ be _maxFraction_ - max(_fractionDigits_, _minFraction_).
```

Comment:
Thought: This could be pulled out into the caller; we can set minFraction to max(fractionDigits, minFraction) at an earlier point in the spec

Reply:
Yes, but the logic needs to be defined somewhere. Also, for significant digits you need to determine the count of integer digits to determine the cut point, and that's not known before calling ToRawPrecision().